src/pages/registration/contributors_tab/components/AddContributorModalContent.tsx

```diff
@@ -61,6 +71,14 @@ const AddContributorModalContent = ({
   const [searchTerm, setSearchTerm] = useState(initialSearchTerm);
   const debouncedSearchTerm = useDebounce(searchTerm);
   const [authorities, isLoadingAuthorities] = useFetchAuthorities(debouncedSearchTerm);
+  const user = useSelector((store: RootStore) => store.user);
+
+  const { values } = useFormikContext<Registration>();
+  const {
+    entityDescription: { contributors },
+  } = values;
+
+  const isSelfAddedAsAuthor = contributors.some((contributor) => contributor.identity.id === user?.authority?.id);
```

Comment:
kan det bli kluss her hvis vi har en ikke-verifisert contributor og brukeren ikke har authority? undefined = undefined?
men det gjør i så fall kanskje ikke noe heller, siden man da ikke vil få opp knappen som gir mening om man ikke har authority

Reply:
hvis dette er er problem holder det sikkert å legg til en sjekk mot null her, og legge til `&& user.authority` på linje 133

Reply:
godt poeng. legger til den sjekken 👍  trenger vel ikke å sjekke mot null hvis vi uansett sjekker på user.authority?

Reply:
nei, men litt forvirrende at `isSelfAddedAsAuthor` kan være true om brukeren ikke har autority (for da kan man jo ikke være lagt til)

Reply:
men user?.authority?.id = user && user.authority && user.authority.id, så når jeg tenker meg om så trenger vi jo ikke den sjekken allikevel? right?

Reply:
`?` blir jo `null` hvis det er tomt da, så hvis `contributor.identity.id` også er `null` for contributors som ikke er verifisert blir det jo kluss, men er ikke sikker på hva det blir

Reply:
hvis man ikke har authority skal man vel ikke kunne åpne wizard uansett(?), så mulig det ikke er relevant heller

Reply:
men `undefined` er ulik `null` da. og `contributor.identity.id` vil vel være undefined for unverified authors?